test/integration/model.elements.api.test.ts

```diff
@@ -137,11 +160,11 @@ describe('Registry API - retrieve Model Bpmn elements', () => {
         target: 'gateway_with_flows_id',
       });
       expectStartEvent(modelElements[2] as ShapeBpmnSemantic, {
-        eventDefinitionKind: ShapeBpmnEventDefinitionKind.NONE,
         id: 'start_event_in_adHoc_sub_process_id',
         name: 'Start Event In AdHoc Sub-Process',
         outgoing: ['sequence_flow_in_adHoc_sub_process_1_id'],
         parentId: 'expanded_adHoc_sub_process_id',
+        eventDefinitionKind: ShapeBpmnEventDefinitionKind.NONE,
```

Comment:
**nitpick:** extra changes. Properties are sorted in alphabetical order.

Reply:
I change the order to be consistent with the other tests.   
Also, I don't understand why we should order them by alphabetical. with this order we don't see the important and mandatory properties.

Reply:
The definition of what is an "important" property is subjective, while alphabetical order is just an easier convention to follow.